script.js

```diff
@@ -1,3 +1,26 @@
-'use strict';
+console.log(this === window);
+// returns true
 
-debugger;
\ No newline at end of file
+function checkThis(){
+  'use strict';
+  console.log(this);
+}
+checkThis();
+// outputs Window object
+//
+function Chair(style, color) {
+  console.log(this);
+  this.style = style;
+  this.color = color;
+}
+sofa = new Chair("sofa", "green");
+
+var couch = {
```

Comment:
I can do that. Since it's not a code-along or lab I would generally leave the work in in the ruby labs but I can clear it